pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -553,6 +553,7 @@ func TestExpectationsOnRecreate(t *testing.T) {
 	}
 
 	f.Start(ctx.Done())
+	f.WaitForCacheSync(ctx.Done())
```

Comment:
Actually, for this one it is done one line below.

Reply:
oh right didn't notice. thanks! closing the PR then